packages/usa-footer/src/styles/_usa-footer.scss

```diff
@@ -66,6 +65,7 @@ $-chevron-expand-more: map-merge(
 
 .usa-footer__primary-container {
   @include grid-container($theme-footer-max-width);
+  @include u-padding-x(0);
```

Comment:
> [!note]
> This is the only meaningful change from #6093. All other changes are just re-ordering the style rules to match convention.

Reply:
I think this looks good to me! Both issues are fairly related, so getting both done in one pull makes sense to me